src/message.ts

```diff
@@ -441,20 +441,21 @@ export class Message implements Sayable {
       return contactList
     }
 
-    // define magic code `8197` to identify @xxx
+    // define magic code `8197` to identify @xxx (wechat version > 6.5, old version use normal blank `32`)
     const AT_SEPRATOR = String.fromCharCode(8197)
```

Comment:
What if we received a message from Wechat version < 6.5?

We should be able to deal with that message as well.